.github/workflows/e2e.go.tag.main.config-ldflags-assets-draft-tag.slsa3.yml

```diff
@@ -0,0 +1,127 @@
+on:
+  schedule:
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # # triggers only if push new tag version, like `v0.8.4` or else
+
+permissions: read-all
+
+env:
+  GH_TOKEN: ${{ secrets.E2E_GO_TOKEN }}
+  ISSUE_REPOSITORY: slsa-framework/slsa-github-generator
+  # WARNING: update build job if CONFIG_FILE changes.
+  CONFIG_FILE: .github/configs-go/config-ldflags-tag.yml
+  DEFAULT_VERSION: v34.0.0
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+      - id: create
+        run: |
+          set -euo pipefail
+
+          ./.github/workflows/scripts/e2e-create-release.sh
```

Comment:
I think you need to update this file to mark the release as a draft in the API call.